proteobench/modules/dda_quant/module.py

```diff
@@ -29,9 +30,7 @@ def generate_intermediate(
         """Take the generic format of data search output and convert it to get the quantification data (a tuple, the quantification measure and the reliability of it)."""
 
         # Summarize values of the same peptide using mean
-        quant_raw_df = filtered_df.groupby(["peptidoform", "Raw file"]).mean()[
-            "Intensity"
-        ]
+        quant_raw_df = filtered_df.groupby(["peptidoform", "Raw file"]).Intensity.mean()
```

Comment:
Selects the correct numeric column as the previous behaviour of discarding non-numeric columns was depreciated in `pandas=2`